src/rust/bitbox02-rust/src/hww/api/bitcoin.rs

```diff
@@ -50,6 +57,47 @@ fn coin_name(coin: pb::BtcCoin) -> &'static str {
     }
 }
 
+/// Processes an xpub api call.
+async fn xpub(
```

Comment:
Would be nice to have a simple test for each XPubType, valid input => valid output.

Reply:
unfortunately the encoding happens in C, and we can only mock C calls in Rust unit tests.